plugins/modules/zos_operator.py

```diff
@@ -35,10 +35,28 @@
     default: false
   debug:
     description:
-      - Return debugging information.
+      - Return rexx debugging information.
     type: bool
     required: false
     default: false
+  security:
+    description:
+      - Call command with (verbose) operator which returns security information.
+    type: bool
+    required: false
+    default: false
+  delay:
+    description:
+      - Set maximum time in seconds to wait for response.
+    type: int
+    required: false
+    default: 0
+  reset:
+    description:
+      - Reset maximume time in seconds after action is called.
```

Comment:
That's actually a problem... we can't see the original time setting in REXX... the first time the variable is non-null is after we set it.  (Sort of a '1-way' variable, if you remember those from peek and poke days...). The reset was to set the delay back to a value after the address call.  We don't see and effect from this in the EC's, because Rexx is stood up and shut down on the call, but this might help a shared environment. 

I hate my 'e' key...

Reply:
@richp405  I see, I don't know enough about what I am going to ask so maybe we could lure Mike Fulton (@mikeFultonDevn) in for comment. 

So, I can see why we can't query our zVMs as they are set up and torn down each time (news to me) but on a production system this is not likely the behavior, since we really can't bank on the variable being set or queried as you point our in our zVMs case, then I might suggest some additional doc and also renaming `reset` to pair up with `delay`, something like `delay_reset`, `reset_delay`, `restore_delay`..... 

Follow up question 1: What happens if we set DELAY in a system that does not get torn down, does the value change globally or just per user instance as we had talked before on and believe we found that SET DELAY only impacts the spawned process on z/OS thus not needing a `reset`?

Follow up question 2: What happens in an environment where its torn down like our zVMs, does it matter if we don't reset it, guessing not. If so, we might want to add some to add some doc around these concepts to better educate users with their application.

Reply:
The value should change for the user...if I'm reading right, the rexx address space is tied to the calling user.  That was why I thought we should offer a post-change-change... "set it to 7, then back to 2" as an example.  When the value is changed, it is that value until changed again or torn down.  There is an optional value in a config file that can initialize the delay value (BPXPRM I think), so I'm not sure if we don't get a value because ours is at default, or if that's a quirk of rexx.

Reply:
As I understand it after discussions with others we removed the RESET because each user will get an unique REXX address space to run in thus there is no reason to RESET but i do have another question:

Question:  "That said, if the user saves the rexx with the updated value, then all subsequent invocations, by this user or another, will use the new, saved, value." , **so when we "SET DELAY" it will stick around for subsequent executions**?

If i run with SET DELAY 120 sec, it will remain that way indefinitely ? If so , how would they RESET it short of running a script, any command we can doc ? (not suggesting we offer a RESET just so they can reset it back) but maybe instruction.

Reply:
@ddimatos If Onno meant 'save the Rexx script', that's what should happen.  If he meant 'save the values', like. BPXPRM, again, that should happen.  He worded it a little oddly, but I don't think he meant it would maintain a value from run to run, since each run gets a new stand up/initialization of the environment.